operator/api/v1/csiscaleoperator_types.go

```diff
@@ -39,7 +39,7 @@ type CSIScaleOperatorSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Attacher Node Selector",xDescriptors="urn:alm:descriptor:com.tectonic.ui:advanced"
 	AttacherNodeSelector []CSINodeSelector `json:"attacherNodeSelector,omitempty"`
 
-	// clusters is a collection of GPFS CSICluster properties for the CSI driver to mount.
+	// clusters is a collection of spectrum scale cluster properties for the CSI driver to mount.
```

Comment:
Let's have initials in caps like `Spectrum Scale` ? as it is a product name.

Reply:
Fixed.